groups/live-windows-virtual-machines/bus_obj1.tf

```diff
@@ -0,0 +1,344 @@
+      to_port     = 56745
+      protocol    = "tcp"
+      cidr_blocks = "172.16.202.81/32"
+    },
+    {
+      from_port   = 57021
+      to_port     = 57021
+      protocol    = "tcp"
+      cidr_blocks = "172.16.202.82/32"
+    },
+    {
+      from_port   = 58269
+      to_port     = 58269
+      protocol    = "tcp"
+      cidr_blocks = "172.16.200.196/32"
+    },
+    {
+      from_port   = 58868
+      to_port     = 58868
+      protocol    = "tcp"
+      cidr_blocks = "172.16.200.196/32"
+    },
+    {
+      from_port   = 59351
+      to_port     = 59351
+      protocol    = "tcp"
+      cidr_blocks = "172.16.202.82/32"
+    },
+    {
+      from_port   = 59635
+      to_port     = 59635
+      protocol    = "tcp"
+      cidr_blocks = "172.16.202.81/32"
+    },
+    {
+      from_port   = 59966
+      to_port     = 59966
+      protocol    = "tcp"
+      cidr_blocks = "172.24.9.53/32"
+    },
+    {
+      from_port   = 62243
+      to_port     = 62243
+      protocol    = "tcp"
+      cidr_blocks = "172.24.9.53/32"
+    },
+    {
+      from_port   = 62341
+      to_port     = 62341
+      protocol    = "tcp"
+      cidr_blocks = "172.16.202.81/32"
+    },
+    {
+      from_port   = 63108
+      to_port     = 63108
+      protocol    = "tcp"
+      cidr_blocks = "172.16.202.82/32"
+    },
+    {
+      from_port   = 63712
+      to_port     = 63712
+      protocol    = "tcp"
+      cidr_blocks = "10.80.10.48/32"
+    },
+    {
+      from_port   = 49208
+      to_port     = 65370
+      protocol    = "tcp"
+      cidr_blocks = "172.16.200.9/32"
+    },
+
+  ]
+
+  ingress_with_self = [
+    {
+      rule = "oracle-db-tcp"
+      self = true
+    },
+    {
+      from_port = 6400
+      to_port   = 6400
+      protocol  = "tcp"
+      self      = true
+    },
+    {
+      from_port = 6410
+      to_port   = 6410
+      protocol  = "tcp"
+      self      = true
+    },
+    {
+      from_port = 49163
+      to_port   = 65528
+      protocol  = "tcp"
+      self      = true
+    },
+  ]
+
+  egress_rules = ["all-all"]
+}
+
+# ------------------------------------------------------------------------------
+# Business Objects Server 1 CloudWatch
+# ------------------------------------------------------------------------------
+
+resource "aws_cloudwatch_log_group" "bus_obj_1" {
+  for_each = local.bus_obj_1_cw_logs
+
+  name              = each.value["log_group_name"]
+  retention_in_days = lookup(each.value, "log_group_retention", var.default_log_group_retention_in_days)
+  kms_key_id        = lookup(each.value, "kms_key_id", data.aws_kms_key.logs.arn)
+
+  tags = merge(
+    local.default_tags,
+    map(
+      "Name", "${var.application}-bus-obj-1-server",
+      "ServiceTeam", var.ServiceTeam
+    )
+  )
+}
+
+# ------------------------------------------------------------------------------
+# Business Objects Server 1 EC2
+# ------------------------------------------------------------------------------
+
+module "bus_obj_1_ec2" {
```

Comment:
No userdata passed in for config e.g. cw logs. These being configured manually post build?

Reply:
These are windows servers so will be configured manually post build.